README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# My name is Alia Ai-Ming Tehzeeb Shahzad. 
+
+I'm a **third year** at the College.
+
+I'm double-majoring in *public policy* and *creative writing* inshallah. 
+
+[This](https://www.youtube.com/watch?v=7D3Ud2JIFhA) is my favorite album. 
+
+![This](https://3d4igz27oxtl2iwox73y9smh-wpengine.netdna-ssl.com/wp-content/uploads/2018/07/scottie-dogs-panting.jpg) is the dog I want when I grow up.
```

Comment:
I also had some trouble with inserting an image, so props to you for getting it to show up! (LOVE the dog btw) 

That being said, it looks like the "this" doesn't show up in the rendered version. Maybe it's because you use an internet link, so it's kind fo combining the format for images and the format for links?

Reply:
Good catching this Emma! For some reason, Github does not render captions when using the Markdown format, so, unfortunately, we are not able to see this.